pyuvdata/uvdata.py

```diff
@@ -3082,23 +3482,32 @@ def read(self, filename, axis=None, file_type=None,
 
     def get_ants(self):
         """
-        Returns numpy array of unique antennas in the data.
+        Get the unique antennas that have data associated with them.
+
+        Returns
+        -------
+        ndarray of int
+            Array of unique antennas with data associated with them.
         """
         return np.unique(np.append(self.ant_1_array, self.ant_2_array))
 
     def get_ENU_antpos(self, center=None, pick_data_ants=False):
         """
         Returns antenna positions in ENU (topocentric) coordinates in units of meters.
 
-        Parameters:
-        -----------
-        center : bool, if True: subtract median of array position from antpos
-        pick_data_ants : bool, if True: return only antennas found in data
-
-        Returns: (antpos, ants)
-        --------
-        antpos : ndarray, antenna positions in TOPO frame and units of meters, shape=(Nants, 3)
-        ants : ndarray, antenna numbers matching ordering of antpos, shape=(Nants,)
+        Parameters
+        ----------
+        center : bool
+            if True, subtract median of array position from antpos
```

Comment:
capitalize "If" (and below, and in returns)

Reply:
Thanks!